{"schemaVersion":{"patch":0,"major":0,"minor":3},"kind":"symbol","sections":[],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/digimecore\/sdkerror\/healthdataerrorerror(error:)"]}],"metadata":{"fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"healthDataErrorError","kind":"identifier"},{"text":"(","kind":"text"},{"text":"error","kind":"externalParam"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"text":")","kind":"text"}],"symbolKind":"case","modules":[{"name":"DigiMeCore"}],"title":"SDKError.healthDataErrorError(error:)","roleHeading":"Case","role":"symbol","externalID":"s:10DigiMeCore8SDKErrorO015healthDataErrorG0yACs0G0_p_tcACmF"},"hierarchy":{"paths":[["doc:\/\/me.digi.sdk.example.librarycore.DigiMeCore\/documentation\/DigiMeCore","doc:\/\/me.digi.sdk.example.librarycore.DigiMeCore\/documentation\/DigiMeCore\/SDKError"]]},"identifier":{"url":"doc:\/\/me.digi.sdk.example.librarycore.DigiMeCore\/documentation\/DigiMeCore\/SDKError\/healthDataErrorError(error:)","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Authorization failed with the specified error"}],"primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["iOS"],"tokens":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"healthDataErrorError"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"error"},{"text":": ","kind":"text"},{"text":"Error","kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP"},{"text":")","kind":"text"}]}],"kind":"declarations"}],"references":{"doc://me.digi.sdk.example.librarycore.DigiMeCore/documentation/DigiMeCore/SDKError/healthDataErrorError(error:)":{"url":"\/documentation\/digimecore\/sdkerror\/healthdataerrorerror(error:)","type":"topic","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"healthDataErrorError","kind":"identifier"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"error"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":")"}],"title":"SDKError.healthDataErrorError(error:)","kind":"symbol","abstract":[{"type":"text","text":"Authorization failed with the specified error"}],"role":"symbol","identifier":"doc:\/\/me.digi.sdk.example.librarycore.DigiMeCore\/documentation\/DigiMeCore\/SDKError\/healthDataErrorError(error:)"},"doc://me.digi.sdk.example.librarycore.DigiMeCore/documentation/DigiMeCore":{"title":"DigiMeCore","url":"\/documentation\/digimecore","type":"topic","kind":"symbol","role":"collection","identifier":"doc:\/\/me.digi.sdk.example.librarycore.DigiMeCore\/documentation\/DigiMeCore","abstract":[]},"doc://me.digi.sdk.example.librarycore.DigiMeCore/documentation/DigiMeCore/SDKError":{"kind":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"SDKError","kind":"identifier"}],"abstract":[{"text":"SDK Errors","type":"text"}],"type":"topic","role":"symbol","navigatorTitle":[{"text":"SDKError","kind":"identifier"}],"url":"\/documentation\/digimecore\/sdkerror","title":"SDKError","identifier":"doc:\/\/me.digi.sdk.example.librarycore.DigiMeCore\/documentation\/DigiMeCore\/SDKError"}}}